script/lib/config.py

```diff
@@ -81,8 +81,8 @@ def product_name():
 
 
 def get_chrome_version():
-    version = (os.environ.get('npm_config_brave_version') or
-               brave_browser_package()['config']['projects']['chrome']['tag'])
+    version = (os.environ.get('npm_config_chrome_version') or
+               brave_core_package()['config']['projects']['chrome']['tag'])
```

Comment:
this was missed when Chromium tag got moved from b-b

Reply:
looks like this is used by bump_version script

Reply:
reverted rename of `npm_config_brave_version` to `npm_config_chrome_version`